parquet-encoding/src/main/java/org/apache/parquet/bytes/BytesInput.java

```diff
@@ -261,10 +270,16 @@ private SequenceBytesIn(List<BytesInput> inputs) {
     @Override
     public void writeAllTo(OutputStream out) throws IOException {
       for (BytesInput input : inputs) {
-        if (DEBUG) LOG.debug("write " + input.size() + " bytes to out");
-        if (DEBUG && input instanceof SequenceBytesIn) LOG.debug("{");
+        if (LOGGER.isDebugEnabled()) {
+          LOGGER.debug("write " + input.size() + " bytes to out");
+        }
+        if (LOGGER.isDebugEnabled() && input instanceof SequenceBytesIn) {
```

Comment:
Can this be nested in the last if statement instead of calling `isDebugEnabled` twice?